tests/backend/test_ledgercomm.py

```diff
@@ -56,24 +57,35 @@ def test_receive_ok(self):
                 rapdu = self.backend.receive()
         self.check_rapdu(rapdu, payload=payload)
 
+    def test_receive_ok_raise(self):
+        failure, payload = 0x8000, b"something"
+        with patch("ledgercomm.transport.HID") as mock:
+            self.hid = mock
+            self.hid().recv.return_value = (failure, payload)
+            with self.backend:
+                self.backend.set_raise_policy(RaisePolicy.RAISE_ALL)
+                with self.assertRaises(ExceptionRAPDU) as error:
+                    self.backend.receive()
+        self.assertEqual(error.exception.status, failure)
+
     def test_receive_nok(self):
         failure, payload = 0x8000, b"something"
         with patch("ledgercomm.transport.HID") as mock:
             self.hid = mock
             self.hid().recv.return_value = (failure, payload)
             with self.backend:
+                self.backend.set_raise_policy(RaisePolicy.RAISE_NOTHING)
                 rapdu = self.backend.receive()
         self.check_rapdu(rapdu, status=failure, payload=payload)
 
-    def test_receive_nok_raises(self):
+    def test_receive_nok_raise(self):
```

Comment:
To clarify why there was a 's' at `raises`, the tests is named as:
- `test_` - alright it's a test
- `receive` - the checked method name or member
- `nok` - we're testing a non-nominal behavior
- `raises` - the expected behavior: the method **raises** - like in `self.assertRaises` (most time it is not a verb, in this case it is)

As a coding habit it is not better or worse than any other habit, but I find unnecessary to have these cosmetic changes in a PR dealing with other stuff.